pkg/scalers/gcp/gcp_stackdriver_client.go

```diff
@@ -20,7 +21,12 @@ import (
 )
 
 const (
-	defaultTimeHorizon = "1m"
+	// Although the "common" value could be 1m
+	// before v2.13 it was 2m, so we need to
+	// keep that value to not break the behaviour
+	// We need to revisit this in KEDA v3
+	// https://github.com/kedacore/keda/issues/5429
+	defaultTimeHorizon = "2m"
```

Comment:
Do you really think this is such a breaking change that shall be respected? Isn't this more like an improvement to set it from 2 to 1?

Reply:
The problem is that it's failing to some users. I'd prefer to keep the previous behaviour and supporting to change it.
I don't have any strong opinion, but I guess that we should respect the previous value to prevent a breaking change